docs/diagrams/EditExecuteRefrence.puml

```diff
@@ -0,0 +1,57 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+!include Styles.puml
+
+box model COLOR_MODEL_SEGMENT
+participant ":Stock" as Instrument COLOR_MODEL
+end box
+
+activate Instrument
+group sd check if parameter exist in HashMap and edit
+
+Instrument ->Instrument: editGeneralParameter({"name, new name"})
+activate Instrument
+note left: This method contains editName, editCurrentPrice,\neditSentiment and editDoneStatus,\nwhich are similar methods.
```

Comment:
Remove this note as we shld show all the edit method calls in editGeneralParameter